code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -120,16 +120,16 @@
 	if(eats_mice && isturf(loc) && !incapacitated())
 		for(var/mob/living/simple_animal/mouse/M in view(1, src))
 			if(!M.stat && Adjacent(M))
-				custom_emote(1, "splats \the [M]!")
+				custom_emote(1, "splats [M]!")
 				M.death()
 				M.splat()
 				movement_target = null
 				stop_automated_movement = 0
 				break
 		for(var/obj/item/toy/cattoy/T in view(1, src))
-			if(T.cooldown < (world.time - 400))
-				custom_emote(1, "bats \the [T] around with its paw!")
-				T.cooldown = world.time
+			if(!TIMER_COOLDOWN_CHECK(src, "cat_toy"))
```

Comment:
The intention here was to avoid making a whole new variable on `/mob/living/simple_animal/pet/cat` just for cat toys since that's an option now. Although that said there's already an `eats_mice` variable there, so the `world.time` ones would actually work better for this.